### PR TITLE
THRIFT-4026 TSSLSocket doesn't work with Python < 2.7.9

### PR DESCRIPTION
**Pull request by nsuke**

### CODE REVIEW

**Comment by jeking3** [comment]
Build job 17 has been failing due to a haskell issue fairly regularly.  I don't know how to fix it, but if we can clear it out it would be nice. :)

File: build/docker/centos6/Dockerfile [hunk 1]
```diff
@@ -41,7 +41,9 @@ RUN yum install -y epel-release && \
     && yum clean all
 
 # optional dependencies
-RUN pip install ipaddress backports.ssl_match_hostname tornado
+# skipping ipaddress and backports.ssl_match_hostname to test legacy callback
+# RUN pip install ipaddress backports.ssl_match_hostname tornado
+RUN pip install tornado
```
Comment (jeking3) [inline build/docker/centos6/Dockerfile]: Does the ubuntu docker build need to be modified to match?

**Comment by tsuna** [comment]
LGTM, thanks for the fix!

**Comment by jeking3** [comment]
I fixed the build job #17 issue with THRIFT-4044 so you can merge this or rebase to get a clean build first.

**Pull request closed without merging**